config/crd/bases/apps.foundationdb.org_foundationdbclusters.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.6.1
```

Comment:
this seems wrong 🤔 
I am not sure why kubebuilder version was downgraded after make clean all.